src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -59316,7 +59316,7 @@ interface I2
             var compilation2 = CreateEmptyCompilation(source2, options: WithNonNullTypesTrue(TestOptions.ReleaseDll), references: new[] { compilation1.EmitToImageReference(), MinCorlibRef });
 
             compilation2.VerifyEmitDiagnostics();
-            CompileAndVerify(compilation2);
+            CompileAndVerify(compilation2, verify: Verification.Fails);
```

Comment:
>verify: Verification.Fails [](start = 43, length = 26)

Consider adding a comment to indicate why verification is expected to fail.

Reply:
Will add a comment in this PR. Going to start another though that makes this unnecessary. In the same way we skip verification for netstandard because we know it will fail we should skip verification for non-mscorlib corlibs unless specifically asked to do verification. This is a non-trivial change so going to pull it out.